hls4ml/utils/config.py

```diff
@@ -321,6 +323,77 @@ def config_from_pytorch_model(
     model_config['Strategy'] = 'Latency'
 
     config['Model'] = model_config
+    config['PytorchModel'] = model
+    config['InputShape'] = input_shape
```

Comment:
I would add a check if the passed input shape makes sense. Later on if it doesn't it's not easy to figure out why.

Reply:
I have been thinking about that. In Pytorch it seems like the exact input shape is not determined by the model architecture, so it's not possible to completely infer it during parsing. We we can still check some general features, like the number of dimensions of the input, based on the type of the first layer. I'll implement something along those lines.

Reply:
I actually meant much simpler, just check if input shape is a list/iterable and not `None`. Because technically you can pass `None` and you'll get a strange error later. Since this is one of the top user-facing functions, it makes sense we do some user input validation here so we have more confidence later

Reply:
Ah, got it. I changed it to enforce that the input shape is a tuple for a single input or a list of tuples for multiple inputs.